data/phones/list_phones.py

```diff
@@ -51,6 +55,42 @@ def _pick_examples_for_display(examples: Set[str]) -> List[str]:
     return sample
 
 
+def _check_ipa_phonemes(
+    phone_to_examples: Dict[str, Set[str]], args: argparse.Namespace
+):
+    """Given the phonemes checks whether they are represented in IPA.
+
+    This will catch problematic phonemes, such as `ú` which are not valid
+    according to the current IPA standard supported by `ipapy`. In addition, it
+    is likely to complain about highly specific allophones, which are likely
+    to be present in languages which have highly phonetic representation of
+    their phoneme inventory. For current IPA chart please see:
+      https://www.internationalphoneticassociation.org/IPAcharts/IPA_chart_orig/IPA_charts_E.html
+    """
+    bad_ipa_phonemes = set()
+    for phone in phone_to_examples.keys():
+        if not ipapy.is_valid_ipa(phone):
+            bad_ipa_phonemes.add(phone)
+
+    logger = logging.getLogger(__name__)
+    if len(bad_ipa_phonemes) and args.filepath.endswith("phonemic.tsv"):
+        logger.warning(f"Found {len(bad_ipa_phonemes)} invalid IPA phonemes:")
```

Comment:
As per the interpolation point earlier, this should read:

```python
logging.warning("Found %d invalid IPA phones:", len(bad_ipa_phonemes)`
```

If loglevel is above warning-level, the interpolation will never be performed.

Reply:
Done.